### PR TITLE
Postmortems POC

### PR DESCRIPTION
**Pull request by yanjost**

### CODE REVIEW

File: src/_posts/security/postmortems/2023-10-19-SPM-2023-001.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+During Cloudflare outage, `1.1.1.1` randomly returned `SRVFAIL DNS` errors. Ideally, our system should have detected this and switched to using `8.8.8.8`. However, this didn't happen, the errors were not detected, causing 2 of our 4 load balancers to be configured with an empty list of routers.
+
+Each of the load balancers is assigned to one of our public IP addresses, meaning that 50% of the incoming traffic on the `osc-secnum-fr1` region was no longer processed. Due to the current configuration of our monitoring system, we did not receive any alerts about this event.
+
+As a quick remedial solution, we made `8.8.8.8` our main DNS resolver for the service discovery process which has brought the platform back to full operating capacity. Later that day, we deployed a fix to prevent updating the router list on a load balancer when the service discovery returns an empty list.  We are also taking steps to detect incidents faster and increase our resilience to DNS resolution disruptions.
+
+![Untitled](https://prod-files-secure.s3.us-west-2.amazonaws.com/2d57f6e0-45a8-4174-9f2c-142d983f5843/27552254-4426-4f7d-bd81-1ce8fb3711f4/Untitled.png)
```
[the remark this answers is not in the document]
Comment (yanjost) [reply src/_posts/security/postmortems/2023-10-19-SPM-2023-001.md:33]: Yes, the picture is missing. I would add it in the documentation repo, no ? (it's not the final document)

[the remark this answers is not in the document]
Comment (yanjost) [reply src/_posts/security/postmortems/2023-10-19-SPM-2023-001.md:33]: @johnsudaar any reason the picture should be in the CDN and not simply in the assets of the documentation ?

[the remark this answers is not in the document]
Comment (johnsudaar) [reply src/_posts/security/postmortems/2023-10-19-SPM-2023-001.md:33]: It should be in both.

From the readme:
> To insert an image, first upload it to our CDN, inside the documentation folder. Give it a public permission Grant public read access to this object(s). Then, insert it with:

**Review by mrunichs — APPROVED** [review]
LGTM

**Pull request merged**